modules/saliency/src/BING/objectnessBING.cpp

```diff
@@ -104,7 +104,7 @@ int ObjectnessBING::loadTrainedModel( std::string modelName )  // Return -1, 0,
 
   if( !matRead( s1, filters1f ) || !matRead( sI, idx1i ) )
```

Comment:
Could you use a constant reference as an argument of `loadTrainedModel`?

Reply:
uh, found [the reason, it wasn't const](https://github.com/berak/opencv_contrib/blob/62331a6ce0f252b363b0db8adc3e4a7540e46348/modules/saliency/src/BING/objectnessBING.cpp#L101)

but since loadTrainedModel() is [only called once](https://github.com/berak/opencv_contrib/blob/62331a6ce0f252b363b0db8adc3e4a7540e46348/modules/saliency/src/BING/objectnessBING.cpp#L387) , and that without any args, i'd rather remove the whole arg.